src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -64,16 +109,6 @@ const DomainFragment = ({
     }));
   }, [unfollowedDomains]);
 
-  React.useEffect(() => {
-    const parseToken = async () => {
-      const id = await getUserId();
-      if (id) {
-        setMyId(id);
-      }
-    };
-    parseToken();
-  }, []);
-
   const __handleOnPressDomain = (item) => {
     const navigationParam = {
       item: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This function navigates to a different screen with some parameters. Ensure that the destination screen can handle these parameters correctly.
```javascript
const __handleOnPressDomain = (item) => {
  const navigationParam = {
    item: {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->